enctests/testframework/utils/outputTemplate.py

```diff
@@ -15,12 +16,23 @@ def _exportGraph(reportconfig, graph, alltests):
   :param graph: The specific graph to output.
   :param alltest: The raw test data to use.
   """
-  df = pd.DataFrame(alltests)
-  df = df.sort_values(by=graph.get("sortby", "name"))
+  graphargs = graph.get("args")
+
+  if "colororder" in graphargs:
+    colororder = graphargs.get("colororder")
+    del graphargs['colororder']
+    for d in alltests:
+      d['colororder'] = colororder.index(d[graphargs['color']])
```

Comment:
```suggestion
    for test in alltests:
      test['colororder'] = colororder.index(test[graphargs['color']])
```
Just a nitpick for readability